diff --git a/mumps/Makefile b/mumps/Makefile
index da175ce..4efad51 100644
--- a/mumps/Makefile
+++ b/mumps/Makefile
@@ -26,6 +26,7 @@ build/lib/libdmumps.a:${MUMPS_TAR_FILE}
 	rm -rf build;
 	tar xzf ${MUMPS_TAR_FILE}
 	mv mumps-${MUMPS_VER}/ build;
+	patch -p1 < ../../../quiet.diff
 	cd build; cp Make.inc/Makefile.inc.generic.SEQ Makefile.inc;
 
 	cd build; echo "CC = "   ${CC}       >> Makefile.inc;
