diff --git a/mumps/Makefile b/mumps/Makefile
index 195529b..da175ce 100644
--- a/mumps/Makefile
+++ b/mumps/Makefile
@@ -32,7 +32,7 @@ build/lib/libdmumps.a:${MUMPS_TAR_FILE}
 	cd build; echo "FC = "   ${FC}       >> Makefile.inc;
 	cd build; echo "FL = "   ${FC}       >> Makefile.inc;
 	cd build; echo "OPTC = " ${CFLAGS}  ${PTHREAD_INCLUDE} >> Makefile.inc;
-	cd build; echo "OPTF = " ${FCFLAGS} ${PTHREAD_INCLUDE}  >> Makefile.inc;
+	cd build; echo "OPTF = " '-I$$(topdir)/libseq' ${FCFLAGS} ${PTHREAD_INCLUDE}  >> Makefile.inc;
 	cd build; echo "OPTL = " ${FCFLAGS} ${PTHREAD_INCLUDE}  >> Makefile.inc;
 	cd build; echo "LPORDDIR = "`pwd`"/PORD/lib/"     >> Makefile.inc;
 	cd build; echo "IPORDDIR = "`pwd`"/PORD/include/" >> Makefile.inc;
